pkg/config/templates/rancherd-15-logging.yaml

```diff
@@ -0,0 +1,95 @@
+resources:
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging
+    releaseName: rancher-logging
+    version: {{ .LoggingChartVersion }}
+    defaultNamespace: cattle-monitoring-system
```

Comment:
defaultNamespace: cattle-monitoring-system ??

Reply:
changed to `cattle-logging-system`